src/backend/base/langflow/base/agents/crewai/crew.py

```diff
@@ -52,10 +52,7 @@ def get_task_callback(
         self,
     ) -> Callable:
         def task_callback(task_output: TaskOutput):
-            if self._vertex:
-                vertex_id = self._vertex.id
-            else:
-                vertex_id = self.display_name or self.__class__.__name__
+            vertex_id = self._vertex.id if self._vertex else self.display_name or self.__class__.__name__
```

Comment:
This simplification, imo, is a bit harder to read in certain cases. Don't mind it enough to change, unless others had opinions too.

Reply:
This is the rule : https://docs.astral.sh/ruff/rules/if-else-block-instead-of-if-exp/
A matter of taste probably...